README.md

```diff
@@ -252,10 +257,9 @@ service studentData on studentServiceListener {
     resource function testError(http:Caller caller, http:Request request) {
         studentData.requestCounts += 1;
         http:Response response = new;
-
         studentData.errors += 1;
         io:println(studentData.errors);
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default ootb system span.
+        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the traces. These tags are added to the default system span.